docs/topics/certificaterotation.md

```diff
@@ -0,0 +1,52 @@
+# Cluster certificate rotation
+
+Instructions on rotating TLS CA and certificates for an AKS Engine cluster.
+
+## Prerequesites
+
+- Kubernetes version 1.8.0 or later is required
+- The etcd members MUST be in a healthy state before rotating the CA and certs.
+- The `_output` directory for your cluster deployment, including a working SSH key for accessing the master nodes and `kubeconfig`.
```

Comment:
Why do we need the `_output` directory? Is it just for these two requirements? This might be better stated:

- A working ssh private key that has root access to all master nodes, and a working `kubeconfig` file in order to establish an authenticated connection to the Kubernetes cluster API. These files are persisted at AKS Engine template generation time, by default to the `_output/` child directory from the working parent directory at the time of the `aks-engine` invocation.

(or something like that)

Reply:
👍